pysam/libcalignedsegment.pyx

```diff
@@ -2448,8 +2442,9 @@ cdef class PileupColumn:
             # warning: there could be problems if self.n and self.buf are
             # out of sync.
             for x from 0 <= x < self.n_pu:
-                pileups.append(makePileupRead(&(self.plp[0][x]),
-                                              self._alignment_file))
+                pileups.append(
+                    makePileupRead(&(self.plp[0][x]),
```

Comment:
p.s. Pretty sure there is no need for parens around `&self.plp[0][x]`, since `&` has lower precedence than `[]`.